### PR TITLE
M28 New composer documentation

### PR DESCRIPTION
**Pull request by samehaidi**

### CODE REVIEW

File: docs/GettingStarted.rst [hunk 1]
```diff
@@ -1695,6 +1695,34 @@ You can add a text message or not and then click on 'Post' button to share the f
 
 .. note:: If the upload is still in progress when you apply the attachments, it will continue in the background. An animation around the attachments counter badge indicates progress. Note that the post button will be disabled until all files are uploaded.
 
+Selecting destination folder for uploads
```
Comment (thomasdelhomenie) [inline docs/GettingStarted.rst]:
```suggestion
Selecting destination folder for uploaded files
```

File: docs/GettingStarted.rst [hunk 2]
```diff
@@ -1695,6 +1695,34 @@ You can add a text message or not and then click on 'Post' button to share the f
...
+``````````````````````````````````````````
+
+You can select destination folders for uploaded files from the new composer.
```
Comment (thomasdelhomenie) [inline docs/GettingStarted.rst]:
```suggestion
You can select the destination folder of the uploaded files in the new composer. 
```

File: docs/GettingStarted.rst [hunk 3]
```diff
@@ -1695,6 +1695,34 @@ You can add a text message or not and then click on 'Post' button to share the f
 
 .. note:: If the upload is still in progress when you apply the attachments, it will continue in the background. An animation around the attachments counter badge indicates progress. Note that the post button will be disabled until all files are uploaded.
 
+Selecting destination folder for uploads
+``````````````````````````````````````````
+
+You can select destination folders for uploaded files from the new composer. 
+Once the files are uploaded, you can use the 'folder' icon to choose a destination folder where to store them.
```
Comment (thomasdelhomenie) [inline docs/GettingStarted.rst]:
```suggestion
Once the files are uploaded, you can use the 'folder' icon to choose the folder in which the will be stored.
```

File: docs/GettingStarted.rst [hunk 4]
```diff
@@ -1695,6 +1695,34 @@ You can add a text message or not and then click on 'Post' button to share the f
...
+It is possible to select the storage for all uploaded files using the folder icon above all files or file per file separately.
+
+|image234|
+
+When you select the principal folder icon, all uploaded files will be stored in the selected folder.
```
Comment (thomasdelhomenie) [inline docs/GettingStarted.rst]:
```suggestion
When you use the principal folder icon, all uploaded files will be stored in the selected folder.
```

File: docs/GettingStarted.rst [hunk 5]
```diff
@@ -1695,6 +1695,34 @@ You can add a text message or not and then click on 'Post' button to share the f
...
+When you select a destination folder for one file, the information will be displayed and it is possible to undo the action using the 'X' icon
+
+|image236|
+
+If the added files are selected from existing uploads, it is not possible to change the destination folder.
```
Comment (thomasdelhomenie) [inline docs/GettingStarted.rst]:
```suggestion
If the attached files are selected from existing uploads, it is not possible to change the destination folder. 
```

File: docs/GettingStarted.rst [hunk 6]
```diff
@@ -1695,6 +1695,34 @@ You can add a text message or not and then click on 'Post' button to share the f
...
+
+|image234|
+
+When you select the principal folder icon, all uploaded files will be stored in the selected folder.
+When you select a destination folder for one file, the information will be displayed and it is possible to undo the action using the 'X' icon
+
+|image236|
+
+If the added files are selected from existing uploads, it is not possible to change the destination folder. 
+A forbidden sign icon is displayed instead of the folder one for these files. A tooltip "Not available for selected files" is displayed when we mouse over this icon.
```
Comment (thomasdelhomenie) [inline docs/GettingStarted.rst]: Not sure that the sentence "A tooltip ..." is relevant. We are going too deep in the detail IMO, and does not bring interesting information to the user.

File: docs/GettingStarted.rst [hunk 7]
```diff
@@ -1695,6 +1695,34 @@ You can add a text message or not and then click on 'Post' button to share the f
...
+
+|image235|
+
+If the added files are a mix between uploaded (by drag and drop or manually) and selected files, the folder selected as a destination folder will be applied only for uploaded files.
```
Comment (thomasdelhomenie) [inline docs/GettingStarted.rst]:
```suggestion
If the attached files are a mix between uploaded (by drag and drop or manually) and selected files, the folder selected as a destination folder will be applied only for uploaded files. 
```

File: docs/GettingStarted.rst [hunk 8]
```diff
@@ -1695,6 +1695,34 @@ You can add a text message or not and then click on 'Post' button to share the f
...
+
+|image235|
+
+If the added files are a mix between uploaded (by drag and drop or manually) and selected files, the folder selected as a destination folder will be applied only for uploaded files. 
+A tooltip "Only applicable for uploaded files" is displayed when you mouse over this icon.
+
+|image237|
+
+.. note:: The default destination folder is the following for:
+Home page composer: /Public/Activity Stream Documents under the Personal Documents drive.
```
Comment (thomasdelhomenie) [inline docs/GettingStarted.rst]:
```suggestion
Home page composer: `/Public/Activity Stream Documents` under the `Personal Documents` drive.
```

File: docs/GettingStarted.rst [hunk 9]
```diff
@@ -1695,6 +1695,34 @@ You can add a text message or not and then click on 'Post' button to share the f
...
+Space composer : /Activity Stream Documents under the space’s drive.
```
Comment (thomasdelhomenie) [inline docs/GettingStarted.rst]:
```suggestion
Space composer : `/Activity Stream Documents` under the space’s drive.
```

**Review by thomasdelhomenie — CHANGES_REQUESTED** [review]

File: docs/GettingStarted.rst (hunk 6 above)
Comment (samehaidi) [reply docs/GettingStarted.rst]: Ok i deleted information about tooltips

**Review by thomasdelhomenie — APPROVED** [review]

**Pull request merged**